understanding/non-text-content.html

```diff
@@ -178,82 +167,57 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
                   
-                  <dt>Goal</dt>
-                  <dd>Non-text information is available to more people.</dd>
+                  <dt>目標</dt>
+                  <dd>非テキストコンテンツをより多くの人々が利用できるようにする。</dd>
```

Comment:
さすがに「information」を「コンテンツ」としてしまうのは苦しいのでは…？
```suggestion
                  <dd>非テキスト情報をより多くの人々が利用できるようにする。</dd>
```